src/implementations/twig/components/link/link.story.js

```diff
@@ -26,86 +25,125 @@ const withParagraph = (story) => {
   return typeof demo === 'string' ? storyAsString(demo) : storyAsNode(demo);
 };
 
-const iconsList = {};
-iconsList.none = '';
-
-uiIcons.forEach((icon) => {
-  iconsList[icon] = icon;
-});
+const iconsList = uiIcons.reduce((a, b) => {
+  a[b] = b;
+  return a;
+}, {});
+
+const getArgTypes = (data) => {
+  return {
+    label: {
+      name: 'label',
+      type: { name: 'string', required: true },
+      defaultValue: data.link.label,
+      description: 'The main label of the button',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Link',
+      },
+      control: {
+        type: 'text',
+      },
+    },
+    aria_label: {
+      name: 'aria label',
+      type: { name: 'string', required: true },
+      defaultValue: data.link.aria_label,
+      description: 'The aria label attribute',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Accessibility',
+      },
+      control: {
+        type: 'text',
+      },
+    },
+    icon_position: {
+      name: 'icon position',
+      type: { name: 'select', required: false },
+      description: 'Position of the icon (required to show the icon)',
+      table: {
+        type: { summary: 'string' },
+        category: 'Link',
```

Comment:
we should put the icon position in the icon category I guess

Reply:
Questionable, the icon_position is a property of the link, not one belonging to the icon, indeed.

Reply:
for the button, I put everything in the icon category